package/score-worker/src/main/resources/META-INF/spring/score/context/scoreWorkerSchedulerContext.xml

```diff
@@ -13,6 +13,8 @@
         <task:scheduled ref="workerConfiguration" method="refresh" fixed-delay="#{configRefreshInterval}"/>
 		<task:scheduled ref="workerManager" method="logStatistics" fixed-rate="#{statisticsInterval}"/>
         <task:scheduled ref="sessionDataHandler" method="sessionTimeOutScheduler" fixed-rate="60000"/>
+        <task:scheduled ref="workerExecutionMonitorService" method="executeScheduledWorkerMonitors" fixed-delay="#{scheduledWorkerMonitorInterval}"/>
```

Comment:
regarding the schedulers I prefer that the consumer,that uses the data, would run the jobs and not score.